addons/l10n_ar/models/res_partner.py

```diff
@@ -110,3 +110,15 @@ def l10n_ar_identification_validation(self):
                 raise ValidationError(_('Only numbers allowed for "%s"') % rec.l10n_latam_identification_type_id.name)
             except Exception as error:
                 raise ValidationError(repr(error))
+
+    @api.model
+    def _get_id_number_sanitize(self):
```

Comment:
We also have the l10n_ar_vat/l10n_ar_formatted_vat field, ...  Nothing we can reuse?  It does not help to keep the code simple.